README.md

```diff
@@ -18,6 +18,14 @@ Seb's notes : I also changed the pulse parameters for Samsung, update the Panaso
 5. Restart your Arduino ide
 6. Check out the examples
 
+###### Using Git to install library ( Linux )
+`cd ~/Arduino/libraries
```

Comment:
This [appears](https://github.com/markszabo/IRremoteESP8266/blob/3009dd6e2685ae24dd00d6e05aeb5c443c61eb92/README.md) as a single line to me.
i.e.
`cd ~/blah git clone https//blah.blah`

Either add a return, or a `;` or `&&` between the two commands.